src/react/components/pages/predict/predictResult.tsx

```diff
@@ -164,6 +184,48 @@ export default class PredictResult extends React.Component<IPredictResultProps,
         document.body.appendChild(fileLink);
         fileLink.click();
     }
+    private getItems() {
+        const {tags, predictions} = this.props;
+        const tagsDisplayOrder = tags.map((tag) => tag.name);
+        for (const name of Object.keys(predictions)) {
+            const prediction = predictions[name];
+            if (prediction != null) {
+                prediction.fieldName = name;
+                prediction.displayOrder = tagsDisplayOrder.indexOf(name);
+            }
+        }
+        // not sure if we decide to filter item by the page
```

Comment:
question: I didn't have sample output csv files at hand, but I suppose user would like to see all items of all pages. Because when downloading JSON we also return document-level results. I will follow up with PM.